### DISCUSSION TITLE
Mount app on a path without writing a wrapper script

### DISCUSSION THREAD

**Discussion by robrwo** [mojolicious/mojo#1840] [Q&A] [answered]
After writing an app, the spec has been changed to have the entire app mounted on a specific path, i.e. all URLs will be prefixed with a path like "/app"

There's a Mount plugin, but this looks like it requires a mini application as a wrapper.

Is there a simpler way to do this, but modifying the routes in the existing application to simply "embed" the defined routes?

**Comment by daleif** [comment, marked as answer]
I have most of mine running as old fashioned CGIs. I use the ResuestBase plugin, credit https://stackoverflow.com/questions/23571608/configure-urls-with-prefix-in-mojolicious-behind-reverse-proxy-proxypass/52211430#52211430, with some path setup in my apache.

Works just fine. The app has no idea what outer URL it is using.

**Reply by robrwo** [reply]
That works nicely, thanks.
